packages/jekyll-yaml/0.1.0/README.md

```diff
@@ -0,0 +1,25 @@
+This package lets you add some yaml-metadata to a jekyll post.
+
+Typing `:newpost` yields the following:
+
+```
+---
+date: 2024-12-05
+author: YourName
```

Comment:
How about adding a note about editing the name value in line 8 of `package.yml`, for Espanso newbies? Or make it a `global_vars:` value nearer the top of the file?

Reply:
that makes it much prettier, thanks and done